integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/CoinbaseServicesViewModel.kt

```diff
@@ -69,8 +69,16 @@ class CoinbaseServicesViewModel @Inject constructor(
     val activePaymentMethodsFailureCallback = SingleLiveEvent<Unit>()
     val coinbaseLogOutCallback = SingleLiveEvent<Unit>()
 
+    private val _latestUserBalance: MutableLiveData<String> = MutableLiveData()
+    val latestUserBalance: LiveData<String>
+        get() = _latestUserBalance
+
     private fun getUserAccountInfo() = viewModelScope.launch(Dispatchers.Main) {
-        _showLoading.value = true
+        if(config.lastCoinbaseBalance.isNullOrEmpty()) {
+            _showLoading.value = true
+        }else{
+            _latestUserBalance.value = config.lastCoinbaseBalance
+        }
```

Comment:
So we will show the Loading dialog the first time the user views this Fragment?

Reply:
in case for any reason the user did not get the balance on Service screen